amr-wind/wind_energy/actuator/turbine/fast/turbine_fast_ops.H

```diff
@@ -243,7 +243,10 @@ struct InitDataOp<TurbineFast, SrcTrait>
         amrex::ParallelDescriptor::Comm_dup(
             amrex::ParallelDescriptor::Communicator(), tdata.tcomm);
 
-        int sz_info[4]{0, 0, 0, 0};
+        int sz_info[5]{0, 0, 0, 0, 0};
+        // Check if the size of sz_info is 5
+        assert(sizeof(sz_info) / sizeof(sz_info[0]) == 5);
```

Comment:
```suggestion
        AMREX_ALWAYS_ASSERT(sz_info.size()) == 5);
```

Reply:
TBH I would `git checkout main -- amr-wind/wind_energy/actuator/turbine/fast/turbine_fast_ops.H` having made sure to pull the latest main before. I don't think we should see any diffs in this file.

Reply:
I agree with @marchdf , this file should not be changed by this PR. The reason this shows up as a conflict is because I started working on it with the fix I had on my local branch before @marchdf 's PR was merged. And because I needed this for an older version of OpenFAST.

Reply:
Marc's method is best, but I think if you get the merge conflict out of the way, you'll be able to update the branch through GitHub and that'll clear the diffs. I can go ahead and do that